### DISCUSSION TITLE
note_contents

### DISCUSSION THREAD

**Discussion by GuShaocheng** [TriliumNext/Trilium#3661] [General]
### Describe feature

Hi developer, 
When I access documents.db via sqlite3 by
SELECT * FROM note_contents
It seems that I got all notes. But most of them are unreadable. I thought it needs to be decoded in some way.
Could you give me an idea to decode my notes? 
Or it maybe is somewhere, could you guide me to find the document? 
Best.

**Comment by zadam** [comment]
There's no special encoding (unless you protect/encrypt them) - e.g. text notes are just HTML.

It's possible that your client thinks it's binary data and doesn't want to display the contents, perhaps.
